### DISCUSSION TITLE
Qual Odoo usar?

### DISCUSSION THREAD

**Discussion by rubensmalheiro** [OCA/l10n-brazil#2683] [Q&A]
Olá a todos, bem estou a 1 mês tentando decidir qual Odoo usar, hoje tenho um erp na Omie porém gostaria muito de migrar tudo para Odoo por causa de outras coisas além do modulo contabilidade.

Acabei assinando a versão online até descobrir que aparentemente não atende todo o mercado Brasil ou minhas necessidades.

Como estou a decidir o que eu faço acabei descobrindo vocês que alias estão de parabéns pelo que fazem pela comunidade com as adaptações brasileiras espero poder ajudar no futuro.

Bem minha duvida principal seria qual versão do Odoo devo usar ? A comunity? a enterprise a 14 como vi na mensagem anterior ou a 16?

O que esta acontecendo na versão que estou usando é que não consigo de jeito nenhum acertar os impostos e penso se no futuro seria possível a emissão de notas fiscais automáticas.

E alguem tem alguma opnião pessoal sobre comunity vs enterprise?

Espero não ter atrapalhado e nem sei se aqui seria o melhor lugar para postar esta duvida, porém estou a semanas tentando achar um lugar e esse aqui pareceu ser o melhor caminho.

**Comment by rvalyi** [comment]
Ola @rubensmalheiro definitivamente a não ser que vc quiser acabar preso numa versão fechada hyper cara de alguma empresa preste a quebrar a cara (uns 80% dos ~70 parceiros oficiais da Odoo SA dos 10 ultimos anos não ficaram nem 3 anos no mercado), a sua unica opção para acertar na parte fiscal no Brasil é o Odoo v14 com os modulos da OCA (80% é feita por antigos parceiros da Odoo SA como a gente). Vc pode usar Odoo Enterprise tb, em tese deve ser compatível na maior parte, mas não vai te adiantar nada a respeito do uso no Brasil.

Daqui alguns meses deveria ser possível vc tambem iniciar um projeto na versão 16.0 com a localização da OCA (hoje falta migrar muitos módulos que ainda recebem correções importantes na 14.0). Eh provável que a versão 17.0 seja pulada pelos autores da localização ou que um suporte minimo so iria acontecer daqui uns 10 meses (é muito trabalho adaptar a localização para uma determinada versão, por isso a gente tende a pular as versões impares), na duvida melhor vc não se arriscar numa v17 e ir de v14 ou então de v16 daqui alguns meses.

Agora vc tem que botar algo na sua cabeça: usar o Odoo no Brasil é um desafio muito grande, mesmo com a localização da OCA. A Localização é a mais complexa do mundo apenas e muda a forma como o Odoo funciona de forma muito extensiva para atender as exigências fiscais do Brasil. Ou seja, aquele "Brasil não é para amador" se aplica totalemente ao Odoo. A tendencia é ficar sempre mais facil, já ta 2x mais facil do que na v12 ou talvez 10x mais facil do que tava na época da v8. Mas não da para dizer que ta fácil implementar sozinho ainda, vai ter que estudar muitos meses ou até anos ou então contratar uma empresa especializada, e olha que nem todas tem muita disponibilidade. Boa sorte.

**Reply by rubensmalheiro** [reply]
Nossa Raphael muito mais muito obrigado mesmo por sua resposta 👍  de nivel.
O lance de ter assinado era imaginando que iria ter suporte ao Brasil ledo engano nesse mês intensivo com o Odoo. 
Como estou com poucos lançamentos estou tentando e me dedicando ao máximo aos estudos e graças a sua reposta ja estou finalizando a implantação da 14, o que me traz um alivio sobre os estudos já que me sentia totalmente burro sem saida na 16 online.
O que me preocupa é justamente pensar que estou pondo a operação da minha empresa em um software que pode me deixar na mão a qualquer momento, mas o que me agrada muito é o fato de integrar vendas, suporte e financeiro além do lance de concentrar o CRM em um só ponto.  
Adorei a sua analogia ao não é para amador, e a unica duvida que ficou para mim é e se a Odoo S/A um dia resolver fechar a versão open o que aconteceria, por isso acabei indo na online mas ja tentando migrar aproveitando que estou agora com o sabado livre para isso :) 

Muito obrigado e como disse espero poder ajudar e contribuir ao maximo  com o OCA Brasil então :) não sou bom em developer mas em infra e linux  pode contar comigo.

**Comment by rubensmalheiro** [comment]
Queria agradecer ao @rvalyi instalei a versão comunity hj 14 no ubuntue o módulo fiscal da OCA BRasil. E a todos os desenvolvedores que trabalham aqui no projeto parabéns ao concluir a instalação tudo realmente é deiferente, outro Odoo praticamente um sistema a parte. Porém não é possível usar o módulo de chamado e nem o módulo de contabilidade na versão comunity correto?

Bem irei continuar meus estudos mas realmente fiquei confuso agora o que fazer :)  E quem sabe buscar uma empresa especializada ou consultor para ver se realmente vale a pena ;)

**Reply by mileo** [reply]
Você precisa colocar seu usuário no grupo view "full accounting features".

**Reply by antoniospneto** [reply]
Bem vindo a comunidade @rubensmalheiro 

O módulo da contabilidade é o módulo nativo "account" esse já faz parte da versão community da Odoo SA.
Todo o core da contabilidade e do financeiro está contido nesse módulo.

Algumas coisas que não estão presente você pode complementar com os módulos da OCA.

Um exemplo é esse módulo https://github.com/OCA/account-financial-tools/tree/14.0/account_menu que habilita o menu que dá acesso a varias funcionalidade da contabilidade que ficam "escondidas" quando você não tem apenas versão community instalada.

**Reply by mileo** [reply]
Também provavelmente vai precisar de alguns módulos extras da OCA, da uma olhada nesses projetos: https://github.com/OCA/?q=account&type=all&language=&sort=

**Reply by rubensmalheiro** [reply]
Obrigado Antonio irei seguir suas sugestões e irei conhecer mais a fundo :) Ai ai que curva de aprendizado :)

**Reply by rubensmalheiro** [reply]
Só para registrar :) deu certo @antoniospneto consegui habilitar o modulo account >:) agora terei o fluxo de caixa :) 
Agora é interessante né por que venho do caminho ao contrário de muitos eu acabei conhecendo pela versão paga e agoar estou testando a comunity e quando não tinha o fluxo estava descartando completamente continuar ja que nao versão online não teria o módulo fiscal e isso me deixaria na mão, agora irei pesar também os outros modulos como chamados e relacionamento com clientes que não teria na versão community. Muito obrigado pela ajuda até aqui agora go nos testes
